item.go

```diff
@@ -27,7 +28,7 @@ type Item struct {
 	GUID             string     `xml:"guid"`
 	Title            string     `xml:"title"`
 	Link             string     `xml:"link"`
-	Description      string     `xml:"description"`
+	Description      string     `xml:"itunes:summary"` //`xml:"description"`
```

Comment:
i think this is incorrect.  ISummary below on line 44 of the new file handles the use cases around different summaries.  description is different.

either provide the itunes documentation stating that summary == description, or please revert this line.